features/docs/writing_support_code/hooks/README.md

```diff
@@ -26,28 +25,16 @@ Multiple hooks of the same type are executed in the order that they were defined
 If you wish to control this order, use manual requires in `env.rb` - This file is
 loaded first - or migrate them all to one `hooks.rb` file.
 
-## AfterConfiguration and InstallPlugin
+## AfterConfigurInstallPlugin
 
-[`AfterConfiguration`](#afterconfiguration) and [`InstallPlugin`](#installplugin)
-hooks are dedicated to plugins and are meant to extend Cucumber. For example,
-[`AfterConfiguration`](#afterconfiguration) allows you to dynamically change the
-configuration before the execution of the tests, and [`InstallPlugin`](#installplugin)
-allows to have some code that would have deeper impact on the execution.
-
-### AfterConfiguration
-
-**Note:** this is a legacy hook. You may consider using [`InstallPlugin`](#installplugin) instead.
-
-```ruby
-AfterConfiguration do |configuration|
-  # configuration is an instance of Cucumber::Configuration defined in
-  # lib/cucumber/configuration.rb.
-end
-```
+[`InstallPlugin`](#installplugin) hook is dedicated to plugins and are meant to
+extend Cucumber. For example, it allows to dynamically change the configuration
+before the execution of tests or to have some code that have some impact on the
```

Comment:
before the execution of tests or to add some code that could impact the execution itself.